src/aspire/source/image.py

```diff
@@ -777,30 +951,30 @@ def save_metadata(self, starfile_filepath, batch_size=512, save_mode=None):
         # Get local metadata columns that were added by subclass
         local_cols = self._populate_local_metadata()
 
-        df = self._metadata.copy()
+        metadata = self.get_metadata(as_dict=True, default_value=42).copy()
```

Comment:
42?

Reply:
I htink just double oversight, I'll try removing and using the default

Reply:
Using default of `None`, we did not raise, so I'd like to think that means we're not using this feature.

Reply:
Cool.